### ISSUE TITLE
sql - order by clause used only projection as valid order property.

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#3992]
OrientDB Server v2.0.7

```
select DisplayName as displayName from Node order by DisplayName asc skip 0 limit 20
```

don't works.

```
select DisplayName as displayName from Node order by displayName asc skip 0 limit 20
```

works, but I think the best design is that the original property should be taken or at least both.

Are these issues fixed in release 2.1-RC1 cause the new parser ?

**Comment by StarpTech** [comment]
Tested against 2.1-RC1 and still occurs.

**Comment by luigidellaquila** [comment]
Hi @StarpTech 

this is considered a minor issue and will be addressed in 3.0

**Comment by StarpTech** [comment]
Any updates?

**Comment by luigidellaquila** [comment]
Hi @StarpTech 

It's already fixed in develop branch with the new executor (3.0), but this feature will hardly be back-ported to 2.2, as it's a completely new component

Thanks

Luigi

**Comment by santo-it** [comment]
Hi,

Per latest message from @luigidellaquila , since this has been fixed in develop branch (3.0) and the backport to 2.2 is hard, I'm closing this issue now.

2.0 and 2.1 are EOL

Please feel free to comment in case and we will reopen the issue

Many thanks,
